### PR TITLE
BaSP-TG-57: Implement react hooks form

### PR DESCRIPTION
**Pull request by juliancit0**
In this PR I 

- Modified the "Inputs" and "Select" components to work with react-hooks-form.
- implemented react-hook-forms to all the entities.

### CODE REVIEW

**Review by TomasBettini — DISMISSED** [review]
Good job!

**Review by Juancapp — DISMISSED** [review]

**Review by eialyafe — DISMISSED** [review]
Approved!

File: src/Components/Projects/Form/index.js [hunk 1]
```diff
@@ -8,181 +9,141 @@ import FunctionalButton from '../../Shared/Buttons/FunctionalButton';
...
   useEffect(async () => {
+    dispatch(getEmployees());
     if (isValidId) {
       dispatch(setFetching(true));
       try {
         const response = await fetch(`${process.env.REACT_APP_API_URL}/projects/${id}`);
         const data = await response.json();
-        console.log('project data', data);
-        setProjectBody({
+        console.log(data);
```
Comment (mgriolfo-g) [inline src/Components/Projects/Form/index.js:59]:
```suggestion
        console.log(data);
```
This log should be deleted!

File: src/Components/Projects/Form/index.js [hunk 2]
```diff
@@ -8,181 +9,141 @@ import FunctionalButton from '../../Shared/Buttons/FunctionalButton';
...
+  const changeProject = (data) => {
+    dispatch(editProject(id, data));
   };
 
-  const addEmployee = () => {
-    setEmployees(() => {
-      return [...employees, { employee: {}, rate: 0, role: '' }];
-    });
+  const onSubmit = async (data) => {
+    console.log(data);
```
Comment (mgriolfo-g) [inline src/Components/Projects/Form/index.js:95]:
```suggestion
    console.log(data);
```
Deletion also requested for this one

**Review by pabloschivazappa — APPROVED** [review]
Nice!

**Review by mgriolfo-g — APPROVED** [review]
Amazing 🚀

**Pull request merged**